lnbits/settings.py

```diff
@@ -181,6 +181,13 @@ class LnPayFundingSource(LNbitsSettings):
     lnpay_admin_key: Optional[str] = Field(default=None)
 
 
+class AlbyFundingSource(LNbitsSettings):
+    alby_api_endpoint: Optional[str] = Field(default=None)
+    alby_api_key: Optional[str] = Field(default=None)
+    alby_admin_key: Optional[str] = Field(default=None)
+    alby_invoice_key: Optional[str] = Field(default=None)
```

Comment:
These two are not documented in `.env.example` nor in `docs/guide/wallets.md`.
They are also not present in `lnbits/static/js/components/lnbits-funding-sources.js`.

Reply:
these two need to be remove, they are artifacts of cloning another funding source. I'll remove it.